### PR TITLE
Fixes #466 Add package-lock.json

### LINKED ISSUE #466: Add package-lock.json
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

No `package-lock.json` present.
**Expected behavior:**

`package-lock.json` file has to be commited to version control and **not** to add in `.gitignore`
https://medium.com/@Quigley_Ja/everything-you-wanted-to-know-about-package-lock-json-b81911aa8ab8#1c9a

**Steps to reproduce:**

**Yes, Wanna work on this issue**

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #466 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added `package-lock.json` 
-

@mariobehling @vaibhavsingh97 @ParthS007 @gabru-md Please review. Thanks !!

### CODE REVIEW

**Comment by bhaveshAn** [comment]
Please check https://github.com/fossasia/query-server/issues/466#issue-290586184

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "resolved": "https://registry.npmjs.org/bower/-/bower-1.8.2.tgz",
+      "integrity": "sha1-rfU1KcjUrwLvJPuNU0HBQZ0z4vc="
+    },
+    "ci-info": {
+      "version": "1.1.2",
+      "resolved": "https://registry.npmjs.org/ci-info/-/ci-info-1.1.2.tgz",
+      "integrity": "sha512-uTGIPNx/nSpBdsF6xnseRXLLtfr9VLqkz8ZqHXr3Y7b6SftyRxBGjwMtJj1OhNbmlc1wZzLNAlAcvyIiE8a6ZA==",
+      "dev": true
+    },
+    "husky": {
```
Comment (vaibhavsingh97) [inline package-lock.json]: do we need `husky`?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "resolved": "https://registry.npmjs.org/husky/-/husky-0.14.3.tgz",
+      "integrity": "sha512-e21wivqHpstpoiWA/Yi8eFti8E+sQDSS53cpJsPptPs295QTOQR0ZwnHo2TXy1XOpZFD9rPOd3NpmqTK6uMLJA==",
+      "dev": true,
+      "requires": {
+        "is-ci": "1.1.0",
+        "normalize-path": "1.0.0",
+        "strip-indent": "2.0.0"
+      }
+    },
+    "is-ci": {
```
Comment (vaibhavsingh97) [inline package-lock.json]: same?

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/is-ci/-/is-ci-1.1.0.tgz",
+      "integrity": "sha512-c7TnwxLePuqIlxHgr7xtxzycJPegNHFuIrBkwbf8hc58//+Op1CqFkyS+xnIMkwn9UsJIwc174BIjkyBmSpjKg==",
+      "dev": true,
+      "requires": {
+        "ci-info": "1.1.2"
+      }
+    },
+    "normalize-path": {
```
Comment (vaibhavsingh97) [inline package-lock.json]: same

**Comment by vaibhavsingh97** [comment]
@bhaveshAn Update?

**Comment by bhaveshAn** [comment]
@vaibhavsingh97 This `package-lock.json` was generated automatically. No idea of the lines.
See this https://github.com/npm/npm/blob/latest/doc/files/package-lock.json.md

**Pull request merged**